Tests/ttLib/tables/tables_test.py

```diff
@@ -1,327 +1,298 @@
-        "aots/gsub_context1_boundary_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_expansion_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_lookupflag_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_lookupflag_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_multiple_subrules_f1.otf":		('GDEF','GSUB'),
-        "aots/gsub_context1_multiple_subrules_f2.otf":		('GDEF','GSUB'),
-        "aots/gsub_context1_next_glyph_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_simple_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_simple_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_successive_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_boundary_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_boundary_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_classes_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_classes_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_expansion_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_lookupflag_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_lookupflag_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_multiple_subrules_f1.otf":		('GDEF','GSUB'),
-        "aots/gsub_context2_multiple_subrules_f2.otf":		('GDEF','GSUB'),
-        "aots/gsub_context2_next_glyph_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_simple_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_simple_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_successive_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_boundary_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_boundary_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_lookupflag_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_lookupflag_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_next_glyph_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_simple_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_successive_f1.otf":			('GDEF','GSUB'),
-        "aots/lookupflag_ignore_attach_f1.otf":			('GDEF','GSUB'),
-        "aots/lookupflag_ignore_base_f1.otf":			('GDEF','GSUB'),
-        "aots/lookupflag_ignore_combination_f1.otf":		('GDEF','GSUB'),
-        "aots/lookupflag_ignore_ligatures_f1.otf":		('GDEF','GSUB'),
-        "aots/lookupflag_ignore_marks_f1.otf":			('GDEF','GSUB'),
-    ]
 
-    def __init__(self, methodName):
-        unittest.TestCase.__init__(self, methodName)
-        # Python 3 renamed assertRaisesRegexp to assertRaisesRegex,
-        # and fires deprecation warnings if a program uses the old name.
-        if not hasattr(self, "assertRaisesRegex"):
-            self.assertRaisesRegex = self.assertRaisesRegexp
+ttLibVersion_RE = re.compile(r' ttLibVersion=".*"')
 
-    def setUp(self):
-        pass
 
-    def tearDown(self):
-        pass
+def getpath(testfile):
+    path = os.path.dirname(__file__)
+    return os.path.join(path, "data", testfile)
```

Comment:
You also need to split testfile on '/' and rejoin it.  My code did that in `generate_mti_file_test`. Not the best place, I agree.